### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by quentinleroy**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -337,8 +340,7 @@ Parameter           | Mandatory | Description
 `currency`          | true      | 3-letter alphanumeric uppercase code, according to ISO 4217 standard.
                     |           |
 `passengers`        | true      | Array
-`fist_name`         | true      | First name of the passenger, 1-50-letter alphanumeric string.
-`last_name`         | true      | Last name of the customer, 1-50-letter alphanumeric string.
```
Comment (AlexeyKrasnoperov) [inline source/index.html.md]: But you still send first_name and last_name in the example. Why are these parameters deleted?

File: source/index.html.md [hunk 2]
```diff
@@ -593,31 +595,24 @@ Error Code  | Meaning
...
-      "departure_time": "2017-04-29T14:00",
-      "arrival_time": "2017-04-29T15:30",
+      "departure_time": "2017-05-29T14:00",
+      "arrival_time": "2017-05-29T15:30",
       "duration": 5400,
       "title": "mr",
       "first_name": "Balibalo",
       "last_name": "Bateau",
-      "email": "test@mail.com",
+      "email": "quentin@mail.com",
```
Comment (AlexeyKrasnoperov) [inline source/index.html.md]: It's a real email that belongs to real person.
If you want to make an example email, just use 'example.com' host: `quentin@example.com`
Or change it to your own email if you want it to be real.

File: source/index.html.md [hunk 3]
```diff
@@ -913,24 +919,17 @@ curl  'https://api-demo.distribusion.com/retailers/v3/bookings/tD01cTMGVAK-b7FBz
...
```
Comment (AlexeyKrasnoperov) [inline source/index.html.md]: real email

File: source/index.html.md (hunk 1 above)
Comment (quentinleroy) [inline source/index.html.md]: #vacancy only need passengers[][pax]=1&passengers[][type]=PNOS while #bookings needs names. 

In this case it is vacancy, hence I removed the names.

File: source/index.html.md (hunk 2 above)
Comment (quentinleroy) [inline source/index.html.md]: ok thanks for the tips, will change this asap

File: source/index.html.md (hunk 3 above)
Comment (quentinleroy) [inline source/index.html.md]: Done :)

**Pull request merged**